src/main/java/jssc/SerialPortList.java

```diff
@@ -76,6 +83,36 @@ private SerialPortList() {}
         }
     }
 
+    /**
+     * For unit testing purposes
+     */
+    public static Pattern getPortnamesRegexp() {
```

Comment:
If these variables are needed elsewhere, they can be made public.  It's not normal to use getters/setters for final variables and final variables (you've made it no longer final) but it still appears to as final variables are generally capitalized, so this is a bit of an anti-pattern.